activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -256,6 +256,18 @@ def remove_column(table_name, column_name, type = nil, **options) #:nodoc:
         end
       end
 
+      def remove_columns(table_name, *column_names, type: nil, **options)
```

Comment:
This is an implementation detail and should be hidden from the API documentation, like the other SQLite-specific versions of these methods:

```suggestion
      def remove_columns(table_name, *column_names, type: nil, **options) # :nodoc:
```